stubs/seaborn/seaborn/_core/scales.pyi

```diff
@@ -0,0 +1,100 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Sequence
+from dataclasses import dataclass
+from typing import Any, ClassVar
+from typing_extensions import Self, TypeAlias
+
+from matplotlib.axis import Ticker
+from matplotlib.scale import ScaleBase
+from matplotlib.ticker import Formatter, Locator
+from matplotlib.units import ConversionInterface
+from numpy.typing import ArrayLike, NDArray
+from pandas import Series
+from seaborn._core.typing import Default
+
+TransFuncs: TypeAlias = tuple[Callable[[ArrayLike], ArrayLike], Callable[[ArrayLike], ArrayLike]]
+Pipeline: TypeAlias = Sequence[Callable[[Any], Any] | None]
+
+class Scale:
+    values: tuple[Incomplete] | str | list[Incomplete] | dict[Incomplete, Incomplete] | None
+    def __post_init__(self) -> None: ...
+    def tick(self) -> Self: ...
+    def label(self) -> Self: ...
+    def __call__(self, data: Series[Any]) -> ArrayLike: ...
+
+@dataclass
+class Boolean(Scale):
+    values: tuple[Incomplete] | list[Incomplete] | dict[Incomplete, Incomplete] | None = None
+    def tick(self, locator: Locator | None = None) -> Self: ...  # type: ignore[override]
```

Comment:
hm, I don't see why this override is incompatible, and when I tried a variant of this code in mypy I didn't get any errors.